version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
lb.yaml please :-)

Should be:
    netperf:
      netperf:
        file://.